core/src/test/java/io/questdb/griffin/O3FailureTest.java

```diff
@@ -1961,20 +1957,15 @@ private static void testPartitionedDataAppendOODataNotNullStrTailFailRetry0(
                 sqlExecutionContext
         );
 
-        final String expectedMaxTimestamp = prepareCountAndMaxTimestampSinks(compiler, sqlExecutionContext);
-
         for (int i = 0; i < 20; i++) {
             try {
                 compiler.compile("insert into x select * from append", sqlExecutionContext);
-                Assert.fail();
+                Assert.fail("run " + i);
             } catch (CairoException | CairoError ignored) {
             }
         }
-
         fixFailure.set(true);
 
-        assertXCount(compiler, sqlExecutionContext, expectedMaxTimestamp);
```

Comment:
What was the problem with this assertion? Why did we have to remove it?

Reply:
no!! this was a mistake, amazing eye @puzpuzpuz , restoring this.